--- a/Makefile_org	2024-03-11 22:54:37.926158483 +0900
+++ b/Makefile	2024-03-11 23:35:13.554284520 +0900
@@ -1,3 +1,4 @@
+include ../.env
 DEBUG=0
 FRONTEND_SUPPORTS_RGB565=1
 FORCE_32BIT_ARCH=0
@@ -33,6 +34,18 @@
 endif
 endif
 
+# RGB30 plumOS
+ifeq ($(platform), unix)
+        TARGET := $(TARGET_NAME)_libretro.so
+        fpic := -fPIC
+        SHARED := -shared -Wl,--version-script=link.T -Wl,--no-undefined
+        CFLAGS += -marm -mcpu=cortex-a55 -mfpu=neon-fp-armv8 -mfloat-abi=hard
+        CFLAGS += -fomit-frame-pointer -ffast-math
+        CPU_ARCH := arm
+        MMAP_JIT_CACHE = 1
+        HAVE_DYNAREC = 1
+endif
+
 FORCE_32BIT :=
 
 ifeq ($(FORCE_32BIT_ARCH),1)
